articlequality/feature_lists/nlwiki.py

```diff
@@ -54,6 +54,8 @@
         max(wikitext.revision.content_chars, 1),
     cn_templates,
     cn_templates / max(wikitext.revision.content_chars, 1),
+    infobox_templates,
+    infobox_templates / max(wikitext.revision.content_chars, 1),
```

Comment:
This second feature is probably not necessary but no real harm in it.

Reply:
Alright!